code/modules/mining/lavaland/ash_flora.dm

```diff
@@ -109,6 +109,22 @@
 	regrowth_time_low = 3000
 	regrowth_time_high = 5400
 
+/obj/structure/flora/ash/pumpkin_patch
+	icon_state = "pumpkin-harvest"
+	name = "pumpkin patch vines"
+	desc = "Several pupmkins, a cluster of large and scary plants."
+	harvested_name = "pumpkinless patch vines"
+	harvested_desc = "Pumpkin patch vines without any pumpkins on them currently. They should regrow soon..."
+	needs_sharp_harvest = FALSE
+	harvest = /obj/item/reagent_containers/food/snacks/grown/pumpkin
+	harvest_amount_high = 4
+	harvest_time = 50
+	harvest_message_low = "You rip out a pumpkin from the pumpkin vines like a maniac."
+	harvest_message_med = "You delicately remove a few pumpkins from the pumpkin vines. but only a few of them are well grown."
```

Comment:
```suggestion
	harvest_message_med = "You delicately remove a few pumpkins from the pumpkin vines, but only a few of them are well grown."
```

Reply:
I think it would technically be right if I was British. I'm not though.